config/simbot_config.yaml

```diff
@@ -1,4 +1,4 @@
-laser_: "/scan"
+laser_: "/transformed_scan"
```

Comment:
Overall question:

Is this still necessary? Why do we transform the scan? Can't we let the students consider the raw output?

Reply:
This is because pyros scan is backwards. This is not a variation the students should deal with.

Reply:
But for the sim it can just be called scan

Reply:
The reason I did it like this is because the real robot already uses /scan for the raw scan data, so by using /transformed_scan they can both use the same rviz config